modulefiles/gfsutils_gaeac5.intel.lua

```diff
@@ -3,15 +3,15 @@ help([[
   on the NOAA RDHPC machine Gaea C5 using Intel-2023.1.0.
 ]])
 
-whatis([===[Loads libraries needed for building the UFS Weather Model on Gaea ]===])
+whatis([===[Loads libraries needed for building the UFS Weather Model on Gaea C5]===])
```

Comment:
```suggestion
whatis([===[Loads libraries needed for building the GFS utilities Gaea C5]===])
```

Reply:
@aerorahul I shouldn't miss these. Thank you.